### ISSUE TITLE
final report peer review 

### ISSUE THREAD

**Issue by yunsquared** [s-hellm/4741#15]
The goal of this project was to identify top risk factors to build a predictive model on whether teenagers are at risk of heavy drinking, in order to better inform guidance counselors on how to identify early signs of heavy drinking and intervene when necessary. Features examined include age, gender, race, tobacco use, bullying and drug use. 

positives: 
- I really enjoyed the scope and topic of this project, and feel that the results from the analysis and model can be very informative and also alleviate some of the stigma surrounding teen drinking.  
- The steps taken to ensure the model wasn't overfitting or under-fitting was very thorough, and I appreciate the care taken in splitting the data into training, testing and validation sets!
- The use of Gaussian Copula was very well implemented, especially since it was taught quite late in the course. 
- The random forest model's visualizations were also very informative in communicating the results from the grid search, and overall well implemented 

improvement areas: 
- The data preprocessing was hard to follow and not documented very well. Since roughly 17,000 records were removed because they were missing, it might have made sense to consider imputation. 
- There was a lack of consistency for included features in the models used, which was confusing and may weaken the ability to compare accuracy scores across different models.  I think it would have made sense to have a separate feature selection stage for each model instead of perhaps selecting the top X features and using that set for the model selection process. 
- The order of the report was confusing, and I wish the authors spoke about other imputation methods considered if Gaussian copula never converged,  and thus the results were not usable.  
- I think there could have been greater discussion on fairness, especially since a few protected attributes such as race and gender were included in the model.  For example, including appropriate metrics to measure fairness for the classifiers used. 

Overall, I found the methods, problem space, and findings from the report very interesting!
